src/__tests__/type.js

```diff
@@ -715,3 +715,16 @@ test('should give error if we are trying to call type on an invalid element', as
     `"the current element is of type BODY and doesn't have a valid value"`,
   )
 })
+
+test('should submit a form when ENTER is pressed on input', () => {
+  const {element} = setup(`
+    <form>
+      <input type='text'/>
+    </form>
+  `)
+  const onSubmit = jest.fn()
+  element.addEventListener('submit', onSubmit)
+  userEvent.type(element.querySelector('input'), '{enter}')
+
+  expect(onSubmit).toHaveBeenCalledTimes(1)
```

Comment:
```suggestion
  const handleSubmit = jest.fn()
  const {element} = setup(`<form><input type="text" /></form>`, {
    eventHandlers: {submit: handleSubmit},
  })
  userEvent.type(element.querySelector('input'), '{enter}')

  expect(handleSubmit).toHaveBeenCalledTimes(1)
```

Reply:
Hit the wrong button. This won't work after all...

However, the way it's currently coded doesn't clean up the event handler.... One sec.

Reply:
right I always forget that this setup has this param 🤦  sorry